### PR TITLE
Added bacon ipsum generator to samples.

### PR DESCRIPTION
**Pull request by Shy**

### CODE REVIEW

File: samples/bacon-ipsum/app.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            startWithLoremBox.on("input", function () {
+                if (startWithLoremStatus == 0 ){
+                    startWithLoremStatus =1;
+                }else{
+                    startWithLoremStatus = 0;
+                }
+            })
+
+        })
+    </script>
```
[the remark this answers is not in the document]
Comment (Shy) [reply samples/bacon-ipsum/app.html]: With onValueChanged, do we have a recommended behavior? Either an alert to refresh the page or do we just overwrite the value?

File: samples/bacon-ipsum/app.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" >
+    <title>Bacon Ipsum UI Extension</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.js"></script>
+    <link rel="stylesheet" href="https://contentful.github.io/ui-extensions-sdk/cf-extension.css">
+    <script src="https://contentful.github.io/ui-extensions-sdk/cf-extension-api.js"></script>
```
[the remark this answers is not in the document]
Comment (Shy) [reply samples/bacon-ipsum/app.html]: Updated.

File: samples/bacon-ipsum/app.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (floelhoeffel) [reply samples/bacon-ipsum/app.html]: just override - that is what the web app does too

[the remark this answers is not in the document]
Comment (Shy) [reply samples/bacon-ipsum/app.html]: Cool. Should be good to go then with the change in 679531ddf95258e8bdf80e0b5ddd5f95fd11681e.

**Comment by Shy** [comment]
@floelhoeffel Should be good to go. I don't have merge permissions in this repo, but let me know if I need to do anything else to move forward.

**Comment by floelhoeffel** [comment]
Thanks!!

**Pull request merged**